python/ray/serve/api.py

```diff
@@ -129,21 +128,28 @@ def shutdown() -> None:
 
 @PublicAPI(stability="beta")
 def get_replica_context() -> ReplicaContext:
-    """If called from a deployment, returns the deployment and replica tag.
+    """Returns the deployment and replica tag from within a replica at runtime.
 
     A replica tag uniquely identifies a single replica for a Ray Serve
-    deployment at runtime.  Replica tags are of the form
-    `<deployment_name>#<random letters>`.
+    deployment.
 
     Raises:
         RayServeException: if not called from within a Ray Serve deployment.
 
     Example:
-        >>> from ray import serve
-        >>> # deployment_name
-        >>> serve.get_replica_context().deployment # doctest: +SKIP
-        >>> # deployment_name#krcwoa
-        >>> serve.get_replica_context().replica_tag # doctest: +SKIP
+
+        .. code-block:: python
```

Comment:
eventually we will move off this code-block thing, fyi

Reply:
onto what?